tests/src/e2e.spec.ts

```diff
@@ -54,34 +98,92 @@ test.skip('Check welcome page is displayed and that we are redirected to the mai
          */
+
+        mainWindow.webContents.closeDevTools();
         if (mainWindow.isVisible()) {
           resolve(getState());
         } else mainWindow.once('ready-to-show', () => resolve(getState()));
       });
-    },
-  );
+    });
+    expect(windowState.isCrashed, 'The app has crashed').toBeFalsy();
+    expect(windowState.isVisible, 'The main window was not visible').toBeTruthy();
 
-  await page.screenshot({ path: 'tests/output/screenshots/screenshot-welcome-page-init.png', fullPage: true });
+    await page.screenshot({ path: 'tests/output/screenshots/screenshot-welcome-page-init.png', fullPage: true });
 
-  // wait for the initial screen to be loaded
-  const goToPodmanDesktopButton = page.locator('button:text("Go to Podman Desktop")');
-  // wait for visibility
-  await goToPodmanDesktopButton.waitFor({ state: 'visible' });
+    const welcomeMessage = page.locator('text=/Welcome to Podman Desktop.*/');
+    await playExpect(welcomeMessage).toBeVisible();
+  });
 
-  await page.screenshot({ path: 'tests/output/screenshots/screenshot-welcome-page-display.png', fullPage: true });
+  test('Telemetry checkbox is present, set to true, consent can be changed', async () => {
+    // wait for the initial screen to be loaded
+    const telemetryConsent = page.getByText('Telemetry');
+    expect(telemetryConsent).not.undefined;
+    expect(await telemetryConsent.isChecked()).to.be.true;
 
-  // click on the button
-  await goToPodmanDesktopButton.click();
+    await telemetryConsent.click();
+    expect(await telemetryConsent.isChecked()).to.be.false;
+  });
 
-  await page.screenshot({
-    path: 'tests/output/screenshots/screenshot-welcome-page-redirect-to-dashboard.png',
-    fullPage: true,
+  test('Redirection from Welcome page to Dashboard works', async () => {
+    const goToPodmanDesktopButton = page.locator('button:text("Go to Podman Desktop")');
+    // wait for visibility
+    await goToPodmanDesktopButton.waitFor({ state: 'visible' });
+
+    await page.screenshot({ path: 'tests/output/screenshots/screenshot-welcome-page-display.png', fullPage: true });
+
+    // click on the button
+    await goToPodmanDesktopButton.click();
+
+    await page.screenshot({
+      path: 'tests/output/screenshots/screenshot-welcome-page-redirect-to-dashboard.png',
+      fullPage: true,
+    });
+
+    // check we have the dashboard page
+    const dashboardTitle = page.getByRole('heading', { name: 'Dashboard' });
+    await playExpect(dashboardTitle).toBeVisible();
+  });
+
+  test('Verify main UI elements are present in Status Bar', async () => {
+    await playExpect.soft(page.locator('xpath=.//li//div[@title="Help"]'), 'Help was not found').toBeVisible();
+    await playExpect.soft(page.locator('xpath=.//li//div[@title="Tasks"]'), 'Help was not found').toBeVisible();
+    await playExpect
+      .soft(page.locator('xpath=.//li//div[@title="Share your feedback"]'), 'Share your feedback was not found')
```

Comment:
I don't get why we have 3 tests on the same welcome page.

Reply:
I would that these are three distinguished test case:
1. Welcome screen is shown upon start up. 
2. Telemetry has checkbox checked and can be changed
3. go to podman dekstop will close welcome screen and redirects to dashboard